### ISSUE TITLE
[CoE Starter Kit - BUG] Admin | Sync Audit Logs

### ISSUE THREAD

**Issue by MarcosCostarella** [microsoft/coe-starter-kit#2898]
### Describe the issue

Failed to parse invoker connections from trigger 'manual' outputs. Exception: Could not find property 'headers.X-MS-APIM-Tokens' in the trigger outputs. Workflow has connection references '["shared_admin-5foffice-20365-20management-20api-5f7fc9343cf45734fe"]' with invoker runtime source.

### Expected Behavior

Run a child flow

### What solution are you experiencing the issue with?

Audit Log

### What solution version are you using?

2.9

### What app or flow are you having the issue with?

Admin | Sync Audit Logs

### Steps To Reproduce

this instance run today june the 1st
https://us.flow.microsoft.com/manage/environments/0e68ce59-d191-4557-805d-994aa18746b4/flows/c870ac80-b7ce-5cc0-bf1f-7c667b1fa4f1/runs/08585474950321131638159828131CU162

### Anything else?

The input

{
  "text": "61386482-660c-45b0-9b6b-7b1e9c868281",
  "text_1": "2022-05-25T19:30:53.7242090Z",
  "text_2": "2022-05-25T20:30:53.7242090Z"
}

The output
{
  "code": "InvokerConnectionOverrideFailed",
  "message": "Failed to parse invoker connections from trigger 'manual' outputs. Exception: Could not find property 'headers.X-MS-APIM-Tokens' in the trigger outputs. Workflow has connection references '[\"shared_admin-5foffice-20365-20management-20api-5f7fc9343cf45734fe\"]' with invoker runtime source."
}

**Comment by Jenefer-Monroe** [comment]
I believe your connection to the custom connector got lost. Please see https://github.com/microsoft/coe-starter-kit/issues/1424

**Comment by MarcosCostarella** [comment]
> I believe your connection to the custom connector got lost. Please see #1424

Hi Jenefer,

That was my initial assumption. But all the connections status are connected. I have not been prompted to fix any even if I go to dataverse > connections.
I also tried re importing the solution when I was warned it was the same version and I overrided but, no luck.
![image](https://user-images.githubusercontent.com/106347491/171664520-a7762e11-c3b6-441f-8a18-420800eae13c.png)
![image](https://user-images.githubusercontent.com/106347491/171664772-4374fe8c-75e6-4851-9ed3-53a7f28a9c79.png)

**Comment by Jenefer-Monroe** [comment]
Can you validate that the Run-only user property is set? 
Here is some helper information if you need help on that: [Set flow run-only users properties](https://docs.microsoft.com/power-platform/guidance/coe/faq#set-flow-run-only-users-properties)

**Comment by MarcosCostarella** [comment]
Hi Jenefer, you were right, the office 365 management api was running provided by the user instead of the service account. I am wondering why this changed. Really I was about tu submit an issue within the [Child] Admin | Sync Logs but this started happening. Do you want me to close this thread and open new case for the child flow?
![image](https://user-images.githubusercontent.com/106347491/171878446-dac9581e-0b76-4e05-9161-063432d59f1a.png)

**Comment by MarcosCostarella** [comment]
Thanks. As you will see in the images, the parent flow failed when calling the child flow but, only the first 1 of 61 and last instance 61 of 61. I suspect there is some kind of issue with first start time and last end time.
![image](https://user-images.githubusercontent.com/106347491/171910274-c5f6accb-bdbd-454f-a610-1d8e276022a3.png)
![image](https://user-images.githubusercontent.com/106347491/171910373-ba08cf87-bf2b-4eb9-980d-3ed16bb28e2b.png)
If I check most [Child] Admin | Sync Logs flow failures I found an error with the contentId 
![image](https://user-images.githubusercontent.com/106347491/171911080-01fba8e6-dbe2-4050-9d3a-0a2b318c8908.png)
This is the output
{
  "error": {
    "code": "AF20051",
    "message": "Content requested with the key 20220527135924409117618$20220527135929891069788$audit_general$Audit_General$na0026 has already expired. Content older than 7 days cannot be retrieved."
  }
}

**Comment by Jenefer-Monroe** [comment]
What values do you have for the environment variables outlined here? [Set up the Power Automate flow](https://docs.microsoft.com/power-platform/guidance/coe/setup-auditlog#set-up-the-power-automate-flow)
![image](https://user-images.githubusercontent.com/89584709/171919718-0cfd389c-947c-4626-bc06-1f9881c2d332.png)

**Comment by MarcosCostarella** [comment]
StartTime-Interval current value is 7
StartTime-Unit default value is Day
TimeInterval-Unit default value is Hour
TimeInterval-Interval default is 1
TimeSegments-CountLimit default is 60

**Comment by Jenefer-Monroe** [comment]
Yes one of your chunks just goes back too far for the backend to support. It fails on first and last because there is a clean up at one end to get what was missed. 
You should not need to regularly go back multiple days anyway, you would just be re-fetching the same data over and over as the flow runs every day.

**Comment by Jenefer-Monroe** [comment]
Let me know if I'm missing something but I think you should be fine with the defaults there of 1 day back.

**Comment by MarcosCostarella** [comment]
I see. I can roll back to 1. CoE will still persist previous data? Do I need to turn the flow off/on so that takes variable chance?

**Comment by Jenefer-Monroe** [comment]
We start collecting data when you turn the audit logs on and collect as long as they are running. We store them in the Audit Logs table which is in the Core solution. I hope that helps!

**Comment by Jenefer-Monroe** [comment]
Yes since that is a totally new topic, please post a new issue. 
I will close this one then. Thanks for using CoE
